### ISSUE TITLE
False positive for Consistency Warning

### ISSUE THREAD

**Issue by tlux** [rrrene/credo#660]
### Environment

Credo version: 1.0.5
Erlang/Elixir version: 1.8.0
Operating system: macOS Mojave 10.14.5

### What were you trying to do?

I got the following code snippet in my project.

```elixir
entries
|> Stream.map(&json_library().encode!/1)
|> Enum.join("\n")
```

### Expected outcome

No warning should be emitted.

### Actual outcome

A warning is emitted:

    Consistency
    ┃ 
    ┃ [C] ↗ There are spaces around operators most of the time, but not here.

As it seems, Credo thinks the arity part is actually a division by 1 which leads to a consistency warning being emitted. All other mathematical operations in the project have a spaces around the operators.

**Comment by rrrene** [comment]
@tlux Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍
